ADALiOS/ADALiOS/ADAuthenticationRequest+AcquireAssertion.m

```diff
@@ -145,6 +145,7 @@ - (void)attemptToUseCacheItem:(ADTokenCacheItem*)item
     }
     
     //Now attempt to use the refresh token of the passed cache item:
+    BOOL isMultiresourceRefreshToken = item.multiResourceRefreshToken;
```

Comment:
Before item is passed as an parameter to an async call, we mark down whether it is a MRRT.
Such that we don't need to depend on the assumption that *item* cannot be modified.